### PR TITLE
Add GraphQL docs

### PR DESCRIPTION
**Pull request by matsimitsu**
Just a test to see if we can generate docs based on the graphQL schema, (so far I say we can)

- [ ] Clean up design
- [ ] Run through comments on GraphQL fields and types
- [ ] Add global text about GraphQL + how to use it

Part of https://github.com/appsignal/appsignal-server/issues/2333

### CODE REVIEW

File: source/graphql/query.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Every GraphQL schema has a root type for both queries and mutations.
+
+The root type defines how GraphQL operations begin. It is the entry point to constructing GraphQL queries.
+
+<%- (graphql_query['fields'] || []).each do |field| %>
+##<%= field['name'] %>
+<%= field['description'] %>
+
+<%- if field['args'].any? %>
```
Comment (matsimitsu) [inline source/graphql/query.html.md.erb]: Yes, that's erb inside markdown :P

**Comment by thijsc** [comment]
One thing I'm wondering: People outside of AppSignal won't have access to the graphql schema file. Can they still run middleman without it?

**Comment by thijsc** [comment]
Nevermind, we commit it :-)

**Comment by thijsc** [comment]
I've updated the schema task and the schema itself. It now needs some design love and a general introduction. @wesoudshoorn shall we take a look at this?

**Comment by wesoudshoorn** [comment]
@thijsc and myself have talked about designing this, but we decided that the content needed to be improved first before I could properly "design" this.

**Pull request closed without merging**